static/app/views/settings/projectDebugFiles/externalSources/customRepositories/status.tsx

```diff
@@ -26,12 +26,12 @@ function Status({theme, details, onEditRepository, onRevalidateItunesSession}: P
 
   const {
     pendingDownloads,
-    itunesSessionValid,
+    promptItunesSession,
     appstoreCredentialsValid,
     lastCheckedBuilds,
   } = details ?? {};
 
-  if (itunesSessionValid === false) {
+  if (promptItunesSession) {
```

Comment:
IIUC this is now the actual fix?

Reply:
yep! this is the one spot that needed to be updated to check the correct property from the response.